docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -446,9 +444,123 @@ Where,
+simulant's birth weight following the *maternal* consumption of iron fortified 
+foods.
+
+**Hemoglobin Level**
+
+The effect of iron fortified food consumption on children under 7 years of age 
+was obtained from the Keats et al. systematic review. However, the effect size 
+in this review was reported in standardized mean differences rather than in 
+units of hemoglobin concentration directly. Therefore, we performed a separate 
+meta-analysis of the results included in the Keats et al. review. This 
+meta-analysis is shown below.
+
+.. image:: iron_meta.png
+
+Notably, the above changes in hemoglobin concentration are shown in grams per 
+*deciliter* (dL); however, GBD models hemoglobin concentration as grams per 
+*liter* (L). Therfore, the effect size used in our simulation model should be 
+**+3.0 (95% CI: -0.2, +6.1)** grams of hemoglobin per liter of blood. This 
+effect size should be interpreted as the population mean shift in hemoglobin 
+concentrations among children less than seven years old in LMICs following 
+iron fortification of staple foods. 
+
+.. note:: 
+
+	The confidence interval for this effect size includes the null (0), 
+	indicating that for some individual simulation draws, the intervention 
+	effect may *decrease* population mean hemoglobin concentrations. However, 
+	*on average*, the effect of the intervention will increase population mean 
+	hemoglobin levels.
+
+To model the uncertainty in the estimate, the above mean difference (MD) 
+should be drawn from a normal distribution with mean = 3.0, 2.5\ :superscript:`
+th`-percentile = -0.2, and 97.5\ :superscript:`th`-percentile = 6.1. This 
+distbibution can be created using `SciPy's norm function
+<https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.norm.html>`_
+as follows:
+
+.. code-block:: Python
+
+	from scipy.stats import norm
+
+	# mean and 0.975-quantile of normal distribution for mean difference (MD)
+	mean = 3
+	q_975 = 6.1
+
+	# 0.975-quantile of standard normal distribution (=1.96, approximately)
+	q_975_stdnorm = norm().ppf(0.975)
+
+	std = (q_975 - mean) / q_975_stdnorm # std dev of normal distribution
+
+	# Frozen normal distribution for MD, representing uncertainty in our effect size
+	hb_md_distribution = norm(mean, std)
+
+.. note::
+
+	This distirbution is slightly shifted to the right (0.025 percentile is 
+	equal to -0.1, rather than -0.2) due to rounding approximations in the 
+	meta-analysis for the effect size causing a slightly non-symmetrical 
+	confidence interval around the mean.
+
+**Birth Weight**
+
+The effect of maternal consumption of iron fortified food on infant birth weight was obtained directly from Haider et al. (2013). According to this data source, mean birth weight among babies born to mothers who consumed iron fortified foods had a mean birth weight **15.1 grams (95% CI: 6.0, 24.2)** higher than babies born to mothers who did not consume iron fortified foods. 
```

Comment:
This is not quite right.  The Haider number was a dose-response: +15.1 g per additional 10 mg iron the mother gets.  To get the actual effect size for our model, we need to estimate how much additional iron women are getting from fortified food.

Reply:
Oh, my bad, I copied from the word doc concept model so thanks for checking. Have we already looked into how to do this or is this an outstanding task? @Ninicorn

Reply:
Ok, I just pushed a commit that proposes a strategy to handle this, although there are still a couple of outstanding todos within the documentation.